README.md

```diff
@@ -53,12 +53,12 @@ your own libraries atop ggez.
 
 ### Supported platforms
 
- * Fully supported: Windows, Linux
- * Not officially supported but might work anyway: Mac
+ * Fully supported: Windows, Linux, MacOS
```

Comment:
Why did you add MacOS here and Android, iOS, Web below?

Reply:
wgpu supports Metal natively. I've also tested it on Windows and MacOS. In theory, wgpu + winit should also support Android, iOS and Web but I haven't tested that. AFAICT, the only major blocker here is that gilrs does not have support for Android.

Reply:
Weren't the mac issues winit related

Reply:
What were the issues?

Reply:
There were several. The one that comes to my mind is https://github.com/ggez/ggez/pull/949.
Then there were:
- https://github.com/ggez/ggez/issues/577
- https://github.com/ggez/ggez/issues/577
- https://github.com/ggez/ggez/issues/541
- and perhaps https://github.com/ggez/ggez/issues/900

but we can't say if these are still current.

Reply:
All of these seem to be closed though.

Reply:
Yes, but only because Mac support was officially dropped.

Reply:
I too think that it could work. But neither of us owning a Mac hasn't changed and probabaly will not change in the forseeable future. Theoretically I could try to build a Hackintosh, but to be honest that's just not worth it to me right now.

Reply:
I have a Mac system and I can confirm #577 and #553 are no longer issues. I do notice relatively high CPU usage across all the examples (30%) but the framerate is capped to the refresh rate correctly. I did not notice any jitter, so overall #541 is probably okay too.

Reply:
That's good to hear, thanks for checking! Sadly I think MacOS needs to keep a special not-officially-supported-but-should-work status still, as long as we have no Macs ourselves, since we just can't promise any kind of serious support while not owning one.